src/migration/types.ts

```diff
@@ -0,0 +1,80 @@
+export type FileHash = string
+
+export type V2Collection = {
+  id: string,
+  wearables: V2Wearable[]
+}
+
+export type V2Wearable = {
```

Comment:
The `V2` representation shouldn't be necessary any more, let's convert the assets into `V3` directly